polymetis/polymetis/python/polymetis/utils/continuous_grasper.py

```diff
@@ -206,6 +282,27 @@ def continuously_grasp(self, max_iters=1000):
 
         return total_successes, total_tries
 
+    def set_pregrasp_height(self, pregrasp_height: float):
+        self.pregrasp_height = pregrasp_height
+
+    def set_grasp_height(self, grasp_height: float):
+        self.grasp_height = grasp_height
+
+    def set_planner_dt(self, planner_dt: float):
+        self.planner_dt = planner_dt
+
+    def set_gp_range_upper(self, x: float, y: float, z: float):
+        self.gp_range_upper = torch.Tensor([x, y, z])
+
+    def set_gp_range_lower(self, x: float, y: float, z: float):
+        self.gp_range_lower = torch.Tensor([x, y, z])
+
+    def set_up_time_seconds(self, up_time_seconds: float):
+        self.up_time_seconds = up_time_seconds
+
+    def set_down_time_seconds(self, down_time_seconds: float):
+        self.down_time_seconds = down_time_seconds
```

Comment:
You may want to use the [property decorator](https://www.programiz.com/python-programming/property), or simply have this as initialization parameters (if you don't expect it to change during execution).

Reply:
Done